src/AppBanner/index.tsx

```diff
@@ -135,3 +143,5 @@ export const AppBanner: React.FC<Props> = ({
         </div>
     );
 };
+
+export { AppBanner };
```

Comment:
No strong preference either way but just noting that we could go back to the inline `export const AppBanner` if we wanted. What do you reckon?